### ISSUE TITLE
matplotlib.patches.Rectangle?

### ISSUE THREAD

**Issue by 3f6a** [stevengj/PythonPlot.jl#35]
Is it possible to access `matplotlib.patches.Rectangle` (which is not inside `pyplot`) through this package?

**Comment by LKuhrmann** [comment]
Very late but maybe for others:
You can access it via `PythonPlot.matplotlib.patches.Rectangle`

**Comment by stevengj** [comment]
Yes, `PythonPlot.matplotlib` gives you access to the whole module.
